Xamarin.Forms.Core.UnitTests/Xamarin.Forms.Core.UnitTests.csproj

```diff
@@ -218,7 +219,5 @@
       <LogicalName>Images/crimson.jpg</LogicalName>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="StyleSheets\" />
-  </ItemGroup>
-</Project>
```

Comment:
@rmarinho  I think this needs to be rebased to include this folder, no?

Reply:
uhh nice catch. yes please